sdk/storage/Azure.Storage.DataMovement.Blobs/api/Azure.Storage.DataMovement.Blobs.net6.0.cs

```diff
@@ -54,10 +54,13 @@ public partial class BlobStorageResourceContainerOptions
     public partial class BlobStorageResourceOptions
     {
         public BlobStorageResourceOptions() { }
-        public Azure.Storage.Blobs.Models.AccessTier? AccessTier { get { throw null; } set { } }
-        public Azure.Storage.Blobs.Models.BlobHttpHeaders HttpHeaders { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, string> Metadata { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } set { } }
+        public Azure.Storage.DataMovement.DataTransferProperty<Azure.Storage.Blobs.Models.AccessTier?> AccessTier { get { throw null; } set { } }
```

Comment:
I think you mean, we can't clear AccessTier right? I think we can keep this property and document a limitation since the service doesn't allow us to clear AccessTier.